src/HyperFormula.ts

```diff
@@ -643,7 +643,7 @@ export class HyperFormula implements TypedEmitter {
    *
    * @category Cells
    */
-  public getCellSerialized(cellAddress: SimpleCellAddress): NoErrorCellValue {
+  public getCellSerialized(cellAddress: SimpleCellAddress): RawCellContent {
```

Comment:
Should we consider this as a breaking change to the public API? IMO yes.

Reply:
sure